tests-extension/test/ingress_operator_util.go

```
@@ -0,0 +1,700 @@
+		},
+	}
+	_, err := dc.Resource(ingressControllerGVR).Namespace(operatorNamespace).Create(ctx, ic, metav1.CreateOptions{})
+	return err
+}
+
+func deleteIC(ctx context.Context, dc dynamic.Interface, name string) {
+	_ = dc.Resource(ingressControllerGVR).Namespace(operatorNamespace).Delete(ctx, name, metav1.DeleteOptions{})
+	_ = wait.PollUntilContextTimeout(ctx, pollInterval, longTimeout, true, func(ctx context.Context) (bool, error) {
+		_, err := dc.Resource(ingressControllerGVR).Namespace(operatorNamespace).Get(ctx, name, metav1.GetOptions{})
+		if err != nil {
+			return true, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 191) but it returns nil

(nilerr)